frontend/src/scenes/feature-flags/FeatureFlagSchedule.tsx

```diff
@@ -6,39 +6,49 @@ import {
     LemonTable,
     LemonTableColumn,
     LemonTableColumns,
+    LemonTag,
+    LemonTagType,
 } from '@posthog/lemon-ui'
 import { useActions, useValues } from 'kea'
 import { DatePicker } from 'lib/components/DatePicker'
+import { dayjs } from 'lib/dayjs'
 import { More } from 'lib/lemon-ui/LemonButton/More'
 import { atColumn, createdAtColumn, createdByColumn } from 'lib/lemon-ui/LemonTable/columnUtils'
+import { Tooltip } from 'lib/lemon-ui/Tooltip'
 import { useEffect } from 'react'
 
-import { ScheduledChangeType } from '~/types'
+import { groupsModel } from '~/models/groupsModel'
+import { ScheduledChangeOperationType, ScheduledChangeType } from '~/types'
 
 import { featureFlagLogic } from './featureFlagLogic'
 import { FeatureFlagReleaseConditions } from './FeatureFlagReleaseConditions'
+import { groupFilters } from './FeatureFlags'
 
 const featureFlagScheduleLogic = featureFlagLogic({ id: 'schedule' })
 export const DAYJS_FORMAT = 'MMMM DD, YYYY h:mm A'
 
 export default function FeatureFlagSchedule(): JSX.Element {
-    const { featureFlag, scheduledChanges, scheduledChangeField, scheduleDateMarker } =
+    const { featureFlag, scheduledChanges, scheduledChangeOperation, scheduleDateMarker } =
         useValues(featureFlagScheduleLogic)
     const {
         setFeatureFlagId,
         setFeatureFlag,
+        setAggregationGroupTypeIndex,
         loadScheduledChanges,
         createScheduledChange,
         deleteScheduledChange,
         setScheduleDateMarker,
-        setScheduledChangeField,
+        setScheduledChangeOperation,
     } = useActions(featureFlagScheduleLogic)
+    const { aggregationLabel } = useValues(groupsModel)
 
     const featureFlagId = useValues(featureFlagLogic).featureFlag.id
```

Comment:
keeping consistent, easier to read 😅 

```
const { featureFlag } = useValues(featureFlagLogic)
const featureFlagId = featureFlagId
const aggregationGroupTypeIndex = ....
```

Reply:
The problem is `featureFlag` is already destructured from `featureFlagScheduleLogic`, hence accessing the id here directly with the dot notation 😬

Reply:
ahh, I see, minor nit, you can rename it when destructuring to prevent conflicts

const { featureFlag: originalFlag } = useValues(...)
const id = originalFlag.id

Reply:
oh I didn't know this was possible, thanks!